scripts/load_data.sh

```diff
@@ -66,18 +72,18 @@ done
 ### wait for ingestion to complete
 echo "waiting for ingestion to complete"
 RETRY_TIMES=0
-until [ "$(curl -s -u test:test http://rabbitmq:15672/api/queues/gdi/verified | jq -r '."messages_ready"')" -eq 2 ]; do
+until [ "$(curl -s -u test:test http://rabbitmq:15672/api/queues/gdi/verified | jq -r '."messages_ready"')" -eq 4 ]; do
     echo "waiting for ingestion to complete"
     RETRY_TIMES=$((RETRY_TIMES + 1))
-    if [ "$RETRY_TIMES" -eq 30 ]; then
+    if [ "$RETRY_TIMES" -eq 40 ]; then
         echo "::error::Time out while waiting for ingestion to complete"
         exit 1
     fi
     sleep 2
 done
 
 I=0
-for file in NA12878.bam NA12878_20k_b37.bam; do
+for file in NA12878.bam NA12878.bam.bai NA12878_20k_b37.bam NA12878_20k_b37.bam.bai; do
```

Comment:
```suggestion
for file in ${FILES}; do
    if [ "${file: -4}" = ".bai" ]; then
        file="$(basename "$file" .bai).bam.bai"
    fi
```

Reply:
Very not confident that this is an improvement.

Reply:
it makes it slightly easier to add files if we need to so I think yes, improvement indeed :-)